### PR TITLE
Update and typings

### PR DESCRIPTION
**Pull request by SimonSchick**
- Add typings (no conversion yet, sry)
- Update deps (except eslint (NOPE))
- Update supported node versions
- Simplify test scripts
- Fix some tests not running  (and prevented that from happening again)

### CODE REVIEW

File: typings.d.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
+interface ITesters {
```
Comment (ProbablePrime) [inline typings.d.ts]: Interface the properties?
``
ITester {
 handle:...
 profile:....
}
twitter: ITester
``

?

Comment (SimonSchick) [inline typings.d.ts]: I don't see any advantage in doing that.
In fact, that would make things more complicated.

Comment (JamyDev) [inline typings.d.ts]: I agree with Simon

Comment (ProbablePrime) [inline typings.d.ts]: fariu

**Pull request merged**